src/static/chemiscope.css

```diff
@@ -37,14 +37,120 @@
     z-index: 10;
 }
 
-.chsp-open-viewer-settings {
+.chsp-open-viewer-settings{
     position: absolute;
     top: 5px;
     right: 5px;
     /* Place on top of jsmol applet */
     z-index: 10;
 }
 
+.chsp-circle-button {
+  border-radius: 100%;
+  border: solid thin black;
+  position: absolute;
+  top: 10px;
+  height: 20px;
+  width: 20px;
+  /* Place on top of jsmol applet */
+  z-index: 10;
+}
+
+.vertical-rect {
```

Comment:
where are these used? Since CSS has no concept of scope/namespace, the class names need to be prefixed by `chsp-<use-case>-`

Reply:
These are an artifact and will be removed.